engines/scumm/script_v5.cpp

```diff
@@ -3065,11 +3065,19 @@ void ScummEngine_v5::decodeParseString() {
 						// I.e. in total 22650 frames.
 						offset = (int)(offset * 7.5 - 22500 - 2*75);
 
+						// Add the user-specified adjustment.
+						if (ConfMan.hasKey("loom_playback_adjustment")) {
+							int adjustment = ConfMan.getInt("loom_playback_adjustment");
+							offset += ((75 * adjustment) / 100);
+							if (offset < 0)
+								offset = 0;
+						}
+
 						// Slightly increase the delay (5 frames = 1/25 of a second).
 						// This noticably improves the experience in Loom CD.
 						delay = (int)(delay * 7.5 + 5);
 
-						_sound->playCDTrack(1, 0, offset, delay);
+						_sound->playCDTrack(1, 1, offset, delay);
```

Comment:
Out of curiosity, why is this changing?

Reply:
Because I wanted to be able to play from the very start of the track, i.e. offset 0, but specifying 0 both as offset and loop counter caused it to not play the track at all. See DefaultAudioCDManager::play(). If I understand things correctly, specifying 0 and 1 as loop counter does the same thing?